src/homepageExperience/components/steps/cli/InstallDependencies.tsx

```diff
@@ -10,18 +10,10 @@ import {
 } from '@influxdata/clockface'
 
 import React, {FC, useEffect, useState} from 'react'
-import {useDispatch} from 'react-redux'
-import {getBuckets} from 'src/buckets/actions/thunks'
 import {event} from 'src/cloud/utils/reporting'
 import {keyboardCopyTriggered, userSelection} from 'src/utils/crossPlatform'
 
 export const InstallDependencies: FC = () => {
-  const dispatch = useDispatch()
```

Comment:
@alexjw3 @david-rusnak Is it correct that any state retrieved from `getBuckets()` isn't necessary here?

Reply:
Yes, that is unnecessary!